packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -72,6 +72,14 @@
     "description": "The title for the Flutter licenses page."
   },
 
+  "licensesPackageDetailTextZero": "No licenses",
```

Comment:
Shouldn't we just skip (and not display) any packages that have no licenses at all? That way we don't need to do translation for the zero case. You could update the description to say that it isn't allowed to have a value of zero.